### ISSUE TITLE
paramvector only capturing first value

### ISSUE THREAD

**Issue by tstastny** [ethz-asl/fw_px4_plottools#34]
I dont know why this now suddenly stopped working .. or how it ever worked .. these things were tested before we merged. But it appears to be a bug when I run through it now.
https://github.com/ethz-asl/fw_px4_plottools/blob/050e034fa801aed0a1c4857d4d64fe854115ee31/02_helper_functions/ImportPX4LogData.m#L195-L197

The snippet above can be fixed with the following replacement lines:
```
temp_params = temp_params(:,~isnan(temp_params(1,:)));
paramvector.(param_fields{idx_params}) = ...
   timeseries(temp_params(1,:)', temp_params(2,:)');
```

**Comment by tstastny** [comment]
Also need to convert timestamp to seconds

**Comment by acfloria** [comment]
Fixed by #37
